regression/contracts/assigns_enforce_04/test.desc

```diff
@@ -3,12 +3,6 @@ main.c
 --enforce-contract f1
 ^EXIT=0$
 ^SIGNAL=0$
-^\[f1.\d+\] line \d+ Check that x2 is assignable: SUCCESS$
```

Comment:
Random place for comment: I'd love for the PR description ("This PR adds:

In check_frame_conditions_function: ...) to also be in the body of the commit message. This will ensure that using `git log` is truly meaningful.